collector/cpu_vulnerabilities_linux.go

```diff
@@ -0,0 +1,61 @@
+package collector
+
+import (
+	"fmt"
+
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/procfs/sysfs"
+)
+
+import (
+	"github.com/go-kit/log"
+)
+
+type cpuVulnerabilitiesCollector struct {
+	vulnDesc        *prometheus.Desc
+	totalCntDesc    *prometheus.Desc
```

Comment:
We're moving away from struct vars for the `prometheus.Desc` declarations. Please move these to package vars.